libs/dh/auth/feature-msal/src/index.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license
+ * Copyright 2021 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+export * from './lib/b2c-config';
+export * from './lib/msal-constants';
+export * from './lib/msal.broadcast.service';
+export * from './lib/msal.guard.config';
+export * from './lib/msal.guard';
+export * from './lib/msal.module';
+export * from './lib/msal.service';
```

Comment:
suggestion: Since all of these files are almost a copy from MSAL, let's place them in a "3rd-party" folder, which will hopefully make it more obvious that they will be deleted at some point.